### PR TITLE
ROSCON 2019 Workshop demo

### PR DESCRIPTION
**Pull request by dabonnie**
This PR supports the ROSCON 19 Security Workshop. Specifically:

- Add [launch-ros-sandbox](https://github.com/aws-robotics/launch-ros-sandbox) dependency to Dockerfile
- Add new launch file to sandbox navigation nodes

Note: the sandbox demo specific components are provided in a new directory

### CODE REVIEW

File: configs/sandbox_demo/secure.conf [hunk 1]
```diff
@@ -0,0 +1,13 @@
+source $BYOBU_PREFIX/share/byobu/profiles/tmux
+
+# setup secure session
+new-session -s secure -n diagnostic -d
+send-keys 'glances' Enter
+
+setenv FOO "foo"
```
Comment (thomas-moulard) [inline configs/sandbox_demo/secure.conf:7]: do we still need this one? :D

File: configs/sandbox_demo/secure.conf [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+setenv ROS_SECURITY_ENABLE true
+setenv ROS_SECURITY_STRATEGY Enforce
+setenv ROS_SECURITY_ROOT_DIRECTORY $TB3_DEMO_DIR/keystore
+setenv ROS_SECURITY_LOOKUP_TYPE MATCH_PREFIX
+
+source configs/sandbox_demo/security_workshop_demo_common.conf
```
Comment (thomas-moulard) [inline configs/sandbox_demo/secure.conf]: newline?

File: configs/sandbox_demo/security_workshop_demo_common.conf [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+send-keys 'tree keystore -d' Enter
+
+split-window -h
+send-keys 'ros2 security generate_artifacts -k keystore -p policies/tb3_gazebo_policy.xml'
+
+split-window -v
+send-keys 'env | grep ROS' Enter
+
+# reselect turtlebot window (the one with teleop)
+select-window -t turtlebot
```
Comment (thomas-moulard) [inline configs/sandbox_demo/security_workshop_demo_common.conf]: newline

**Review by thomas-moulard — APPROVED** [review]
This LGTM, a few nits. Feel free to address those as you want and merge by yourself this one.

File: configs/sandbox_demo/secure.conf (hunk 1 above)
Comment (dabonnie) [reply configs/sandbox_demo/secure.conf:7]: As discussed we'll leave it so it reflects the original file :-)

File: configs/sandbox_demo/secure.conf (hunk 2 above)
Comment (dabonnie) [reply configs/sandbox_demo/secure.conf]: Addressed in latest push.

File: configs/sandbox_demo/security_workshop_demo_common.conf (hunk 3 above)
Comment (dabonnie) [reply configs/sandbox_demo/security_workshop_demo_common.conf]: Addressed in latest push.

**Pull request merged**